waspc/data/Generator/templates/sdk/package.json

```diff
@@ -97,6 +97,7 @@
       {=! Public: { type WebSocketDefinition, type WaspSocketData } =}
       {=! Private: [server, sdk] =}
       "./server/webSocket": "./dist/server/webSocket/index.js",
+      "./entities": "./dist/entities/index.js",
```

Comment:
When building the prototype without this export, I got the following error:
```
../../../src/MainPage.tsx:11:27 - error TS2307: Cannot find module 'wasp/entities' or its corresponding type declarations.

11 import type { Task } from 'wasp/entities'
                             ~~~~~~~~~~~~~~~

../../../src/Todo.tsx:2:22 - error TS2307: Cannot find module 'wasp/entities' or its corresponding type declarations.

2 import { Task } from 'wasp/entities'
                       ~~~~~~~~~~~~~~~


Found 2 errors in 2 files.

Errors  Files
     1  ../../../src/MainPage.tsx:11
     1  ../../../src/Todo.tsx:2
```
It seems that TS LSP and VS Code know how to find the correct types and provide IDE support without needing an explicit `exports` definition.
TSC, on the other hand, requires the package to export all symbols the user code references (types or otherwise).